### ISSUE TITLE
Creating CRNN Models?

### ISSUE THREAD

**Issue by mrn1522** [samrere/pytortto#6]
Hello,

I was wondering if it's possible to create a CRNN model for text recognition using PyTortto? I tried to use CTCLoss from pytorch but created issues. 

If not, is it possible for me to copy pytorch CTC loss function and LSTM model and implement manually into PyTortto? Or is there a lot of adjustments that required?

**Comment by samrere** [comment]
Hi,

Currently pytortto don't have modules for RNN. You are very welcomed to implement CTCLoss and LSTM modules!

pytorch calculations are not implemented in python, so I think it won't be possible to copy their implementations. I would suggest you to first understand how an LSTM module works, then try to implement it in numpy&cupy. Finally, you should test against pytorch to make sure your LSTM works correctly.
